src/infra/marketplace/marketplaceCmdRepo.go

```diff
@@ -500,6 +625,27 @@ func (repo *MarketplaceCmdRepo) UninstallItem(
 		}
 	}
 
+	catalogItem, err := repo.marketplaceQueryRepo.ReadCatalogItemBySlug(
+		installedItem.AppSlug,
+	)
+	if err != nil {
+		return err
+	}
+
+	err = repo.uninstallFilesRemoval(deleteDto.InstalledId)
```

Comment:
Why do you pass it as an id and then query the database once again for the catalogItem and installedItem?

Reply:
I followed [your suggestion](https://github.com/speedianet/os/pull/107#discussion_r1657356792), thinking it wouldn't be an issue. In any case, it's indeed better to avoid repeated queries. Done!